src/en/send.md

```diff
@@ -353,3 +341,145 @@ If the request is not successful, the response body is `json`, refer to the tabl
 |`429`|`[{`<br>`"error": "TooManyRequestsError",`<br>`"message": "Exceeded send limits (LIMIT NUMBER) for today"`<br>`}]`|Refer to [service limits](limits.md) for the limit number|
 |`500`|`[{`<br>`"error": "Exception",`<br>`"message": "Internal server error"`<br>`}]`|GC Notify was unable to process the request, resend your notification.|
+
+
+## Sending notifications in bulk
+
+```
+POST /v2/notifications/bulk
+```
+
+Send a batch of notifications, up to 50,000 recipients at a time, for a single template. You can schedule to send notifications up to 4 days in the future.
+
+### Request body
+
+```json
+{
+  "name": "My bulk name",
+  "template_id": "f33517ff-2a88-4f6e-b855-c550268ce08a",
+  "rows": [
+    ["email address", "name"],
+    ["alice@example.com", "Alice"],
+    ["bob@example.com", "Bob"]
+  ],
+  "scheduled_for": "2021-06-08T15:15:00", # optional string
+  "sender_id": "f025b1a9-63af-43e8-b969-627bfe544bba" # optional string
+}
+```
+
+### Arguments
+
+**name (required)**
+
+The `name` of your bulk sending. Used to identify this batch of notifications later on.
+
+<Content :page-key="$site.pages.find(p => p.relativePath === 'en/_arg_template_id.md').key"/>
+
+**rows (required)**
+
+An array of arrays. The first line is the header and should include at least `email address` if you're sending an email template or `phone number` if you're sending a text message template. The other column headers should match placeholder fields of your template.
+
+The following lines should be your recipients' details and should match the order of column headers. You can have between 1 and 50,000 recipients.
+
+#### Optional arguments
+
+**scheduled_for (optional)**
+
+If you want to send your batch of notifications in the future, you can specify a datetime up to 4 days in the future, in [ISO 8601 format](https://en.wikipedia.org/wiki/ISO_8601). Example: `2021-06-08T15:15:00` (UTC time).
+
+**sender_id (optional)**
+
+If you want to use a specific reply-to email address when sending a batch of emails, you can specify a reply-to email address ID.
+
+To find your reply-to email address ID:
+
+1. [Sign in to GC Notify](https://notification.canada.ca/sign-in).
+1. Go to the __Settings__ page.
+1. In the __Email__ section, select __Reply-to email addresses__
+1. Copy the ID of the reply-to email address you want to use
+
+By default, GC Notify will use your default reply-to email address if you don't specify one, or none if you didn't set up one.
+
+**csv (optional)**
+
+If you prefer to pass the content of CSV files instead of rows in the `rows` argument, you can do so. Pass the full content of your CSV file in a key named `csv`. Do not include the `rows` argument.
+
+For example
+
+```json
+{
+  "name": "My bulk name",
+  "template_id": "f33517ff-2a88-4f6e-b855-c550268ce08a",
+  "csv": "email address,name\nalice@example.com,Alice"
```

Comment:
Should there be an n before the email: "alice@example.com" ?

Reply:
Yes, newline character is `\n`